latest_cov_build.py

```diff
@@ -70,3 +71,14 @@
         tar.extractall(path='tools')
 
 os.remove(name)
+
+# Download gcno archive
+index = taskcluster.Index()
+taskId = index.findTask('gecko.v2.mozilla-central.' +
+                        'latest.firefox.linux64-ccov-opt')['taskId']
+url = 'https://queue.taskcluster.net/v1/task/' + taskId + '/runs/0/artifacts/public/build/target.code-coverage-gcno.zip'
```

Comment:
A task might be re-run multiple times, we should make sure we download the artifact from the latest run.
Let's use the latestArtifact URL both here and for chrome-map.json

Reply:
Hi @marco-c. If I use `gecko.v2.mozilla-central.latest.firefox.linux64-ccov-opt` indexPath to find the `taskId` of the build, doesn't word `latest` in the path mean that `taskId`would be of the latest build (https://docs.taskcluster.net/manual/using/artifacts#finding-the-latest-build)? 
If so, all I need to change is to download the latest artifact build with the use of `getLatestArtifact(taskId, name)` method making sure that all downloaded files refer to the same `taskId`?
Thanks.

Reply:
Yes, exactly.

Reply:
And use `getLatestArtifact` for all three artifacts (build, gcno and chrome-map).